extensions/nft-search/src/index.tsx

```diff
@@ -0,0 +1,210 @@
+import { Icon, ActionPanel, Action, List, showToast, Toast } from "@raycast/api";
+import { useState, useEffect, useRef, useCallback } from "react";
+import fetch, { AbortError, Headers } from "node-fetch";
+
+export default function Command() {
+  const { state, search } = useSearch();
+  return (
+    <List
+      isLoading={state.isLoading}
+      onSearchTextChange={search}
+      searchBarPlaceholder="Type @trending to see trending NFT collections or type NFT name..."
+      throttle
+    >
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.name} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.name}
+      subtitle={searchResult.description}
+      accessoryTitle={searchResult.username + ", " + searchResult.one_day_floor_change}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.OpenInBrowser title="Open in Genie" url={searchResult.url} />
+          </ActionPanel.Section>
+          <ActionPanel.Section>
+            <Action.CopyToClipboard
+              title="Copy Contract Address"
+              content={searchResult.address}
+              shortcut={{ modifiers: ["cmd"], key: "." }}
+            />
+            {searchResult.website ? (
+              <Action.OpenInBrowser title="Open Official Website in Browser" url={searchResult.website} />
+            ) : null}
+            {searchResult.twitter ? (
+              <Action.OpenInBrowser
+                title="Open Twitter in Browser"
+                url={"https://www.twitter.com/" + searchResult.twitter}
+                icon="twitter.png"
+              />
+            ) : null}
+            {searchResult.discord ? (
+              <Action.OpenInBrowser title="Open Discord in Browser" url={searchResult.discord} icon="discord.png" />
+            ) : null}
+            {searchResult.ins ? (
+              <Action.OpenInBrowser
+                title="Open Ins in Browser"
```

Comment:
Shouldn't this have the same naming as the other in the list?

Reply:
For this problem, you mean naming convention? `serachResult.instagram` or `title="Open Instagram in Browser"`

Reply:
The title, since the other ones don't have?

Reply:
Resolved